data-processing/entities/terms_v1/extractor.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
+from typing import Iterator
+
+import pandas as pd
+import re
+import json
+
+import pysbd
+
+from common.parse_tex import DEFAULT_CONTEXT_SIZE, EntityExtractor, PlaintextExtractor
+from common.types import SerializableEntity
+
+
+###First convert the aggregate glossary json into a csv, this was done on a seperate file
+
+###Extracting all the terms from glossary csv
+glossary = pd.read_csv("gloss.csv")
```

Comment:
To minimize dependencies, can this be read in with the Python builtin library `csv.reader` instead of `pandas`? Check out this example: https://docs.python.org/3/library/csv.html#csv.reader

Reply:
Also, wasn't the list of terms in JSON? Can you remind me why we decided to make it into CSV instead?